django/test/client.py

```diff
@@ -292,11 +298,12 @@ def _get_path(self, parsed):
         # If there are parameters, add them
         if parsed[3]:
             path += str(";") + force_str(parsed[3])
-        path = uri_to_iri(path).encode(UTF_8)
+        path = unquote_to_bytes(path) if six.PY3 else unquote(path)
+        path = repercent_broken_unicode(path)
         # Under Python 3, non-ASCII values in the WSGI environ are arbitrarily
         # decoded with ISO-8859-1. We replicate this behavior here.
         # Refs comment in `get_bytes_from_wsgi()`.
-        return path.decode(ISO_8859_1) if six.PY3 else path
+        return path.decode('iso-8859-1') if six.PY3 else path
```

Comment:
No, I made it because IMHO this is better style. My understanding is that [these constants](https://github.com/django/django/blob/master/django/core/handlers/wsgi.py#L21) only exist for py2/py3 compatibility. Their use was needed in the `UTF_8` line. The new code only does decoding on py3, so it doesn't need the constants and can just pass the string. This seemed strictly more pythonic to me.

Should I change it back? :)